apps/atomy-q/API/tests/Unit/Support/DecimalStringTest.php

```
@@ -0,0 +1,24 @@
+declare(strict_types=1);
+
+namespace Tests\Unit\Support;
+
+use App\Support\DecimalString;
+use Tests\TestCase;
+
+final class DecimalStringTest extends TestCase
+{
+    public function testNormalizeReturnsNullForNonNumericInput(): void
+    {
+        self::assertNull(DecimalString::normalize('banana', 4));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\DecimalString' in method 'testNormalizeReturnsNullForNonNumericInput'. (undefined)

(StaticAccess)